modules/oc-mirror-updating-cluster-manifests.adoc

```diff
@@ -27,12 +27,17 @@ The `ImageContentSourcePolicy` resource associates the mirror registry with the
 $ oc apply -f ./oc-mirror-workspace/results-1639608409/
 ----
 
-. Apply the release image signatures to the cluster by running the following command:
+. If you mirrored release images, apply the release image signatures to the cluster by running the following command:
 +
 [source,terminal]
 ----
 $ oc apply -f ./oc-mirror-workspace/results-1639608409/release-signatures/
 ----
++
+[IMPORTANT]
+====
+If you are mirroring Operators instead of clusters, you do not need to run `$ oc apply -f ./oc-mirror-workspace/results-1639608409/release-signatures/`. Running that command will return an error, as there are no release-signatures to apply.
```

Comment:
If you do keep the admonition, I think release-signatures might need to be changed to either have backticks like `release-signatures` or to just say "release image signatures":
```suggestion
If you are mirroring Operators instead of clusters, you do not need to run `$ oc apply -f ./oc-mirror-workspace/results-1639608409/release-signatures/`. Running that command will return an error, as there are no release image signatures to apply.
```

Reply:
good call - downgraded to note and substituted with "release image signatures" :)